.travis.yml

```diff
@@ -3,5 +3,8 @@ sudo: false
 node_js:
     - "0.12"
     - 4
+    - 6
+before_script:
+    - npm install typescript@^1.7.3
```

Comment:
We could add this as a `devDependency` and run it instead. https://www.npmjs.com/package/npm-install-peers